setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# setup.sh -- environment bootstrapper for python virtualenv
+
+set -euo pipefail
+
+SUDO=sudo
+if ! command -v $SUDO; then
+	echo no sudo on this system, proceeding as current user
+	SUDO=""
+fi
+
+if command -v apt-get; then
+	if dpkg -l python3-venv; then
+		echo "python3-venv is installed, skipping setup"
+	else
+		if ! apt info python3-venv; then
+			echo package info not found, trying apt update
```

Comment:
specify that you were looking for the python3-env package

Reply:
done